### PR TITLE
SMD-553 - Renaming "extraction" module as "transformation" and changing structure

### PR DESCRIPTION
**Pull request by wjrm500**
### Ticket
[SMD-553](https://dluhcdigital.atlassian.net/browse/SMD-553)

### Change description
Renaming `extraction` module as `transformation` as `extraction` will be considered to mean the process of extracting tables from the original workbook data, while `transformation` represents the process of transforming the data to align with our data model. Filenames linked to rounds use numbers not number words now e.g., `round_1` instead of `round_one`, just so that files order as expected in explorer.

[SMD-553]: https://dluhcdigital.atlassian.net/browse/SMD-553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cyrusdobbs — APPROVED** [review]
LGTM - best to check with @nsnape if this is worth holding off merging until https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/402 has been merged

**Review by Joel-Pearce — APPROVED** [review]
LGTM

**Pull request merged**
